CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### 0.28.1 - January 18, 2022
+
+-   Updated TwilioSyncClient version to resolve connection issue
```

Comment:
We should document in the CHANGELOG that `https://github.com/twilio/cocoapod-specs` should be removed from any existing Podfiles.

Reply:
Good fallout